matRad_calcSigmaIni.m

```diff
@@ -0,0 +1,72 @@
+function [sigmaIni] = matRad_calcSigmaIni(baseData,rays,SSD)
```

Comment:
As said above, it might make sense to use this function generally to determine the initial width of the beam, not only for finesampling.